src/main/java/mate/academy/internetshop/controllers/order/OrderController.java

```diff
@@ -12,7 +12,7 @@
 import mate.academy.internetshop.service.ShoppingCartService;
 
 public class OrderController extends HttpServlet {
```

Comment:
```suggestion
public class CompleteOrderController extends HttpServlet {
```
давай змінимо трохи неймінг щоб було зрозуміліше за що відповідає даний контроллер

Reply:
Зрозумів. Уже перейменував.